customer/src/main/java/no/unit/nva/customer/CustomerDoiHandler.java

```diff
@@ -29,6 +46,18 @@ protected UUID getIdentifier(RequestInfo requestInfo) throws InputException {
                    .orElseThrow(fail -> handleIdentifierParsingError(identifier, fail));
     }
 
+    protected Map<UUID, SecretManagerDoiAgentDao> getSecretsManagerDoiAgent() throws JsonProcessingException {
+
+        var secretAsStringJsonArray = secretsReader.fetchSecret(SECRETS_KEY_AND_NAME, SECRETS_KEY_AND_NAME);
+
+        if (isNull(secretAsStringJsonArray)) {
+            secretAsStringJsonArray = "[]";
+        }
+
+        return Arrays.stream(dtoObjectMapper.readValue(secretAsStringJsonArray, SecretManagerDoiAgentDao[].class))
+                   .collect(Collectors.toMap(it -> toUuid(it.getCustomerId()), it -> it));
```

Comment:
Can the variable be named "it" isn't conventional in Java, and here I guess it's "customer", but may be wrong